src/platform/nRF5/ThreadStackManagerImpl.cpp

```diff
@@ -54,8 +54,8 @@ CHIP_ERROR ThreadStackManagerImpl::InitThreadStack(otInstance * otInst)
     // Initialize the generic implementation base classes.
     err = GenericThreadStackManagerImpl_FreeRTOS<ThreadStackManagerImpl>::DoInit();
     SuccessOrExit(err);
-    err = GenericThreadStackManagerImpl_OpenThread_LwIP<ThreadStackManagerImpl>::DoInit(otInst);
-    SuccessOrExit(err);
+    // err = GenericThreadStackManagerImpl_OpenThread_LwIP<ThreadStackManagerImpl>::DoInit(otInst);
```

Comment:
Should we delete instead of commenting out? Maybe replace with TODO if this needs to be added back

Reply:
We should left the code there and fix the issue. I'll add a TODO comment